packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooser.stories.js

```diff
@@ -17,6 +19,19 @@ const columns = [
 	{ key: 'icon', label: 'Icon', hidden: true, order: 5, locked: true },
 ];
 
+function Layout({ children }) {
```

Comment:
You can integrate this kind of things in the default story declaration with Storybook's decorators